koku/masu/database/sql/openshift/all/reporting_ocpallcostlineitem_project_daily_summary_azure.sql

```diff
@@ -58,7 +57,6 @@ SELECT 'Azure' as source_type,
        max(unit_of_measure) as unit,
        sum(pretax_cost) as unblended_cost,
        sum(project_markup_cost) as project_markup_cost,
```

Comment:
```suggestion
       sum(markup_cost) as project_markup_cost,
```
Did we miss this? Your removing this column here https://github.com/project-koku/koku/pull/5582/files#diff-cf5d772e6a5f47982cf575c5811b97dded58fcaf87d6b2f0dafeee0eeac5b352L509